webauthn-authenticator-rs/src/MacAuthn/.gitignore

```diff
@@ -0,0 +1,9 @@
+.DS_Store
+/.build
+/Packages
+/*.xcodeproj
+xcuserdata/
+DerivedData/
+.swiftpm/config/registries.json
+.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata
+.netrc
```

Comment:
A nested .gitignore is a dark pattern here, can you move the things up to the root one please.